microsoft/PowerToys
# [SvgThumnbialProvider] Single app instance

## Summary of the Pull Request

This fixes the issue where SvgThumbnailProvier.exe hangs in specific conditions.


## PR Checklist

- [x] **Closes:** https://github.com/microsoft/PowerToys/issues/31032


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

 - Open a directory with multiple svg files
 - Make icons larger so thumbnails are shown
 - Enable PT SVG thumbnail provider
 - While thumbnail are being generated start clicking around the SVG files
 - Ensure Thumbnails are still being generated as expected
